packages/devtools_app/lib/src/shared/http/http_request_data.dart

```diff
@@ -115,6 +115,7 @@ class DartIOHttpRequestData extends NetworkRequest {
         'reasonPhrase': _request.response!.reasonPhrase,
         'redirects': _request.response!.redirects,
         'statusCode': _request.response!.statusCode,
+        'queryParameters': _request.uri.queryParameters,
```

Comment:
please update `network_model_test.dart` to include query parameters and check the value in `HttpRequest.general` : https://github.com/flutter/devtools/blob/master/packages/devtools_app/test/network/network_model_test.dart/#L234

And also add a test for `HttpRequest.queryParameters` (for example, here is the test for `HttpRequest.responseHeaders`: https://github.com/flutter/devtools/blob/master/packages/devtools_app/test/network/network_model_test.dart/#L385)